src/compositions/TopBar/TopBar.scss

```
@@ -10,7 +10,17 @@ $component: top-bar;
     height: 3rem;
     padding: 0.5rem;
     background: var(--c-grey);
+    position: relative;
+    z-index: 1;
 
+    &__mobile-menu {
+        position: fixed;
+        left: 0;
+        top: 3rem;
+        width: 100vw;
+        height: calc(100svh - 3rem);
+        z-index: -1;
```

I just added this to development, so you'll have to rebase to get it.

```suggestion
        z-index: -1;
        
        @include for-size(tablet-portrait-up) {
            display: none;
        }
```

---

This code is not yet on the main branch, still on your pr.